### PR TITLE
Added new Feature component (Download App Links)

### PR DESCRIPTION
**Pull request by lazarnikolov94**
Also added a `baseUrl` and `paths` in `tsconfig.json` so we can import our components using `@components/...` instead of `../../../../components/...`

Here's how the new component looks like:

image

### CODE REVIEW

**Comment by lazarnikolov94** [comment]
@hauptrolle check this out. I've added a new component, but also made some changes in the imports. I've added path aliases so we can import our components easily like `@/components/ResizableFrame` instead of jumping to it `../../../components/ResizableFrame`. Also added a `jest.config.json` file to hook everything up. Check it out!

File: jest.config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "moduleDirectories": ["node_modules", "src"],
+  "moduleNameMapper": {
+    "@/components/(.*)": "<rootDir>/src/components/$1",
+    "@/utils/(.*)": "<rootDir>/src/utils/$1"
+  },
+  "setupFilesAfterEnv": ["<rootDir>/setupTests.ts"]
+}
```
Comment (hauptrolle) [inline jest.config.json:8]: We already have some jest config in the `package.json`. It think we just should use 1 place for the config. So either `package.json` or the `jest.config.json`

File: tsconfig.json [hunk 2]
```diff
@@ -16,14 +12,13 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": "./src/",
+    "paths": {
+      "@/components/*": ["components/*"],
+      "@/utils/*": ["utils/*"]
+    }
```
Comment (hauptrolle) [inline tsconfig.json:20]: Good one! 👍  Love these aliases!

File: src/pages/templates/pageSections/features/downloadAppLinks.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          <Link href={"/"} passHref>
+            <a target={"_blank"}>
+              <PlayStoreBadge />
+            </a>
+          </Link>
+        </Stack>
+      </Stack>
+    </Box>
+  );
+}
```
Comment (hauptrolle) [inline src/pages/templates/pageSections/features/downloadAppLinks.tsx:46]: This is a good looking component! 👍

**Review by hauptrolle — CHANGES_REQUESTED** [review]
I just merged #17  😁  So some of the files need to be changed.

**Comment by hauptrolle** [comment]
One more thing. I would categorize the new component inside `hero` instead of `features`.

**Comment by lazarnikolov94** [comment]
> One more thing. I would categorize the new component inside `hero` instead of `features`.

You think so? I was imaging it being used as a feature, below the fold. I'd redesign it if it's being used as a hero. Actually, I can make a hero version of it and then we'd have two 😄

**Comment by lazarnikolov94** [comment]
> I just merged #17  😁  So some of the files need to be changed.

I'll check that out tomorrow 🙌

**Comment by hauptrolle** [comment]
> > One more thing. I would categorize the new component inside `hero` instead of `features`.
> 
> You think so? I was imaging it being used as a feature, below the fold. I'd redesign it if it's being used as a hero. Actually, I can make a hero version of it and then we'd have two 😄

Sounds good to me! Than we have 2! 😂

File: jest.config.json (hunk 1 above)
Comment (lazarnikolov94) [reply jest.config.json:14]: I guess it'd be better if we separate the Jest config into its own file. It's more predictable that way.

**Comment by lazarnikolov94** [comment]
@hauptrolle check out my latest changes now. I've moved the Jest config into its own file, I guess it's more predictable that way. Also I left the new component in the Features category, but I'll create a new version of it more suitable for Hero placement.

**Review by hauptrolle — APPROVED** [review]

**Pull request merged**